meson.build

```diff
@@ -0,0 +1,468 @@
+project('vips', 'c', 'cpp',
+    version: '8.13.0',
+    meson_version: '>=0.56',
+    default_options: [
+        'c_std=c99'
+    ]
+)
+
+version_parts = meson.project_version().split('.')
+version_major = version_parts[0]
+version_minor = version_parts[1]
+version_patch = version_parts[2]
+
+# rules:
+# sources changed: increment revision
+# binary interface changed: increment current, reset revision to 0
+#   binary interface changes backwards compatible?: increment age
+#   binary interface changes not backwards compatible?: reset age to 0
+library_current = 57
+library_age = 15
+library_revision = 0
+library_version = '@0@.@1@.@2@'.format(library_current - library_age, library_age, library_revision)
+
+gnome = import('gnome')
+pymod = import('python')
+pkg = import('pkgconfig')
+
+add_project_arguments('-Drestrict=__restrict', language: 'cpp')
+
+cc = meson.get_compiler('c')
+
+glib_dep = dependency('glib-2.0')
+gio_dep = dependency('gio-2.0')
+gobject_dep = dependency('gobject-2.0')
+gmodule_dep = dependency('gmodule-no-export-2.0')
+expat_dep = dependency('expat')
+
+libvips_deps = [
+    glib_dep,
+    gio_dep,
+    gobject_dep,
+    gmodule_dep,
+    expat_dep,
+]
+
+prefix_dir = get_option('prefix')
+lib_dir = prefix_dir / get_option('libdir')
+module_dir = lib_dir / 'vips-modules-@0@.@1@'.format(version_major, version_minor)
+
```

Comment:
I noticed there's no support for i18n. I don't know how easy it is to move the `po/` machinery to meson.

Reply:
`import('i18n').gettext('localename')` and you're done.

Reply:
Ah, I indeed forgot this part, I've just re-added it, the handling of the po/ machinery is very easy in meson